src/shared/components/post/metadata-card.tsx

```diff
@@ -75,10 +75,14 @@ export class MetadataCard extends Component<
           </div>
         )}
         {this.state.expanded && post.embed_video_url && (
-          <iframe
-            className="post-metadata-iframe"
-            src={post.embed_video_url}
-          ></iframe>
+          <div className="ratio ratio-16x9">
+            <iframe
+              allowFullScreen
+              className="post-metadata-iframe"
+              src={post.embed_video_url}
+              title={"Embedded Video: " + post.embed_title}
```

Comment:
If you add the "Embedded Video" string, there will have to be a [translation made.](https://github.com/LemmyNet/lemmy-translations). Alternatively, you could just make the title `post.embed_title`.

Reply:
Ok, for now I will remove the prefix, but it is probably worth adding in the future as screen readers might find it useful.